scripts/README.md

```diff
@@ -48,3 +48,40 @@ Force push if necessary.
 ### Announce
 
 Follow the instructions in https://mui-org.notion.site/Releases-7490ef9581b4447ebdbf86b13164272d.
+
+## Deploy documentation without a release
+
+Sometimes it is necessary to deploy the selected commit(s) without
+deploying all the changes that have been merged into the main branch
+since the previous release (e.g. publishing a blog post or releasing
+urgent docs updates).
+
+To do so, follow these steps:
+
+1. Add the `material-ui-docs` remote if you haven't done this already:
+
+   ```sh
+   git remote add material-ui-docs https://github.com/mui/material-ui-docs.git
+   ```
+
+2. Check out the `latest` branch from `material-ui-docs` remote:
+
+   ```sh
+   git checkout --track material-ui-docs/latest
```

Comment:
I usually do:

```suggestion
   git reset --h material-ui-docs/latest
```

to not have to care about branches. If I run `git checkout --track material-ui-docs/latest` it tells me that the branch already exists.

Reply:
Good catch, `git checkout` isn't a good solution in this case.
`git reset --h material-ui-docs/latest` has a drawback too - it will reset the branch you're currently on to the `material-ui-docs/latest`, so you might lose the commits on this branch if the branch is local-only.

We can do this instead:
`git switch --detach material-ui-docs/latest`
It will not impact your current branch since it only detaches the HEAD.
After adding more commits, you can push it to `material-ui-docs/latest` with:
`git push material-ui-docs HEAD:latest`

What do you think?